### PR TITLE
[Merged by Bors] - Fix Syncing Simulator

### PR DESCRIPTION
**Pull request by blacktemplar**
## Issue Addressed

NA

## Proposed Changes

Fixes problems with slot times below 1 second which got revealed by running the syncing simulator with the default speedup time.

### CODE REVIEW

**Review by AgeManning — APPROVED** [review]
The changes look fine to me. 

I guess adding the syncing sim to the workflow isn't a bad idea. We didn't include it in the past because it was kind of lengthy and not that stable. 

I think with our new workflows which should speed up CI there's no harm in adding some more sophisticated tests.

File: beacon_node/beacon_chain/src/eth1_chain.rs [hunk 1]
```diff
@@ -98,7 +98,7 @@ fn get_sync_status<T: EthSpec>(
     } else {
         // The number of seconds in an eth1 voting period.
         let voting_period_duration =
-            T::slots_per_eth1_voting_period() as u64 * (spec.milliseconds_per_slot / 1_000);
+            T::slots_per_eth1_voting_period() as u64 * spec.milliseconds_per_slot;
```
Comment (michaelsproul) [inline beacon_node/beacon_chain/src/eth1_chain.rs:101]: wait a sec, what's going on here?

bors r-

Comment (michaelsproul) [reply beacon_node/beacon_chain/src/eth1_chain.rs:101]: Ok, I see. The comment above should _definitely_ be updated though, and maybe the name of the variable. `voting_period_duration_millis`?

Comment (michaelsproul) [reply beacon_node/beacon_chain/src/eth1_chain.rs:101]: Or do we even need to change this if we only support seconds anyway?

Comment (michaelsproul) [reply beacon_node/beacon_chain/src/eth1_chain.rs:101]: Sorry @blacktemplar, just me being paranoid :sweat_smile:

Comment (blacktemplar) [reply beacon_node/beacon_chain/src/eth1_chain.rs:101]: Oh yes, this was me trying to fix slot durations that are not multiple of seconds before I recognized the more fundamental problem of having only `seconds_per_slot` in `YamlConfig`. https://github.com/sigp/lighthouse/pull/2083 superseeds those changes anyway, so I am happy to revert them :).

**Review by michaelsproul — APPROVED** [review]
Awesome, let's merge

bors r+

**Pull request closed without merging**